apps/meteor/app/models/server/models/LivechatInquiry.ts

```diff
@@ -246,6 +243,49 @@ export class LivechatInquiry extends Base {
 		return collectionObj.aggregate(aggregate).toArray();
 	}
 
+	async getSortingQuery(sort: Record<string, -1 | 1>): Promise<Document[]> {
```

Comment:
Instead of duplicating the function, modify where this is used from the base model and use the one you created on the raw one 👀

Reply:
yeah, i chose the lazy solution here :eyes:, fixing